recipe/meta.yaml

```diff
@@ -16,7 +16,9 @@ package:
 
 build:
   number: {{ number }}
-{% if cuda_major == 0 or (target_platform == "linux-ppc64le" and cuda_major_minor != (10, 2)) %}
+{% if cuda_major == 0
+    or (target_platform == "linux-ppc64le" and cuda_major_minor != (10, 2))
+    or (target_platform == "linux-aarch64" and (cuda_major != 11 or arm_variant_type != "sbsa")) %}
```

Comment:
Could we please write this in terms of `(10, 2)` as well? Want to make sure this still works smoothly as future versions come out :)

Reply:
I wasn't thinking before changing... @jakirkham What do you expect? `cuda_major_minor < (11, 0)`?

Reply:
Yeah that seems reasonable. If we make that change, can we please do the same thing for `ppc64le`?

Reply:
Sure.

Reply:
@jakirkham I guess I misunderstood what you meant. For ppc64le let's keep it as is until we add CUDA 11 support for it, is that ok?

Reply:
Yeah no worries. That's reasonable. This is admittedly a bit unrelated. Just was thinking on how we might tidy things while we were here